README.md

```diff
@@ -1,2 +1,21 @@
+Status: Currently in draft and non-binding. 
+
 # Argo Conformance Program
-A standard to ensure interoperability, protect users, and prevent surprises when using commercial offerings with Argo in the name. Currently in draft and non-binding. 
+The Argo community continues to grow dramatically – adding new users, new contributors, and new capabilities. The Argo Conformance Program creates incentives and requirements designed to help the ecosystem grow in a sustainable and user-friendly way. As a diverse set of tools grows around Argo, users can trust that products certified as compliant will interoperate with each other and be up-to-date with the latest open source releases. This ensures a consistent user experience between products and flexibility to choose between vendors.
+
+When users use an Argo certified offering they should be able to trust that offering to work with the open source versions of Argo and that configuration designed for Argo will always work.
+
+Software conformance ensures predictability and helps protect the Argo brand. 
+
+# Goals
+* Guarantee a consistent and predictable user experience
+* Guarantee interoperability, including API compatibility
+* Guarantee conformant products are up-to-date with latest Argo releases 
+* Promote contributions from commercial partners
```

Comment:
What is the reason to change to Promote?

Reply:
Is Encourage better?

Reply:
@edlee2121 for any additional changes please use the code suggest feature in a code review. Which will allow people to discuss the change before merging. 

We actually started with "Encourage" and the group felt that it wasn't strong enough as a goal and that we should basically require contributions to be coming back in some way. It's likely this will take the form of certified offerings to have a maintainer in the Argoproj. 

Given that I think we should go back to "Ensure".

Reply:
```suggestion
* Ensure contributions from commercial partners
```

Reply:
I like "Ensure" as the goal to really benefit the community in a measurable way. Thanks.

Reply:
Sure not sure how to use that feature. Let me revert my commit and resubmit it.